app/routes/auth.js

```diff
@@ -1,6 +1,7 @@
 const express = require('express')
 const router = express.Router()
 const authController = require('../controllers/auth')
+// const isActivated = require('../middleware/activate')
```

Comment:
@devesh-verma If I just uncomment it now, and then if this PR gets merged then when anyone else will try to login then first they will have to activate their account so... it will increase their workload thats why.
NO issue if you want I can update it.
Should I ?